tap-agent/src/agent/sender_allocation.rs

```diff
@@ -34,6 +43,55 @@ use crate::{
     tap::{context::checks::AllocationId, escrow_adapter::EscrowAdapter},
 };
 
+lazy_static! {
+    static ref UNAGGREGATED_FEE_PER_SENDER_X_ALLOCATION: GaugeVec = register_gauge_vec!(
+        format!("unagregated_fee_per_sender_x_allocation"),
+        "Unggregated Fees",
+        &["sender", "allocation"]
+    )
+    .unwrap();
+}
+
+lazy_static! {
+    static ref RAV_VALUE: GaugeVec =
+        register_gauge_vec!(format!("rav_value"), "RAV Updated value", &["allocation"]).unwrap();
+}
```

Comment:
It's missing the sender label.
```suggestion
lazy_static! {
    static ref RAV_VALUE: GaugeVec =
        register_gauge_vec!(format!("rav_value"), "Value of the last RAV per sender-allocation", &["sender", "allocation"]).unwrap();
}
```